lighthouse-core/gather/gather-runner.js

```diff
@@ -163,6 +164,23 @@ class GatherRunner {
   }
 
   /**
+   * Add run warning if running in Headless Chrome.
+   * @param {!Driver} driver
+   * @param {!GathererResults} gathererResults
+   * @return {!Promise<undefined>}
+   */
+  static warnOnHeadless(driver, gathererResults) {
```

Comment:
nit: we just computed UserAgent the step before this, so could eliminate driver dep

Reply:
done